### DISCUSSION TITLE
Scavenge request :)

### DISCUSSION THREAD

**Discussion by SkeletonAce** [Zrips/CMI#6391] [Feature Requests]
Hi, I have a request regarding cmi.command.scavenge

I want to be able to limit what enchantment books are given to only vanilla ones.

I use custom enchantments on my server that I don't want players to be able to obtain the books for.

I also use Epicrename and if I give an item the glow effect without any enchantments it gives a very strange enchanted book back when you scavenge it. Something like `Looting III 6671`

**Comment by tomhmagic** [comment]
I'd like this feature too. I have custom tools like efficacy 10 which I don't want them to scavange
